mk/rabbitmq-dist.mk

```diff
@@ -123,23 +162,32 @@ $(DIST_DIR)/%.ez:
 		&& grep -q '^prepare-dist::' $(SRC_DIR)/Makefile) || \
 		$(MAKE) --no-print-directory -C $(SRC_DIR) prepare-dist \
 		APP=$(APP) VSN=$(VSN) EZ_DIR=$(EZ_DIR)
-	$(verbose) (cd $(DIST_DIR) && $(ZIP) $(ZIP_V) -r $*.ez $*)
+	$(verbose) (cd $(DIST_DIR) && $(ZIP) $(ZIP_V) -r $(notdir $(EZ)) $(notdir $(EZ_DIR)))
```

Comment:
I've added `.SUFFIXES: .ez`, but now it matches `plugins/blah.ez`, not just `blah.ez`, so we'll  have to use `$(notdir ..)` anyway.